README.md

```diff
@@ -318,8 +318,8 @@ It is a great solution for communities and companies wanting to privately host t
 - Chat-ops powered by Hubot: scalable horizontal app integration (early access)
 - Massively scalable hosting and provisioning (beta testing now)
 - Native Cross-Platform Desktop Application [Windows, macOS, or Linux](https://rocket.chat/)
-- Mobile app for iPhone, iPad, and iPod touch [Download on App Store](https://geo.itunes.apple.com/us/app/rocket.chat/id1028869439?mt=8)
-- Mobile app for Android phone, tablet, and TV stick [Available now on Google Play](https://play.google.com/store/apps/details?id=com.konecty.rocket.chat)
+- Mobile app for iPhone, iPad, and iPod touch [Download on App Store](https://itunes.apple.com/us/app/rocket-chat/id1148741252?mt=8)
```

Comment:
`geo.` will localise the AppStore-Preview by redirecting. Shouldn't we keep this?

Reply:
Good catch! Will fix it.